packages/firebase_crashlytics/lib/src/firebase_crashlytics.dart

```diff
@@ -54,21 +54,6 @@ class Crashlytics {
     throw StateError('Error thrown by Crashlytics plugin');
   }
 
-  /// Reports the global value for debug mode.
-  /// TODO(kroikie): Clarify what this means in context of both Android and iOS.
-  Future<bool> isDebuggable() async {
```

Comment:
These still exist on the iOS side.

Reply:
Where do you see that?

Reply:
Oh I see, I'll check that

Reply:
https://github.com/FirebaseExtended/flutterfire/blob/bdb95fcacf7cf077d162d2f267eee54a8b0be3bc/packages/firebase_crashlytics/darwin/Classes/FirebaseCrashlyticsPlugin.m#L94